### ISSUE TITLE
RTL8723BE wifi card working

### ISSUE THREAD

**Issue by baddemanax** [svpcom/wifibroadcast#14]
Hi,

Just to let you know that RTL8723BE is working. I have quite a few packet loss, but I don't think it is software related.
I will let you know about further test.
regards

**Comment by svpcom** [comment]
Have you use it as TX or RX?

**Comment by baddemanax** [comment]
hi,

sorry for late reply

i used it as RX, but i will give it a try as TX too

just a though, could it be possible to stream on 2 different ports ? ( i think of putting 2 cameras on the PI)

________________________________
De : Vasily Evseenko <notifications@github.com>
Envoyé : lundi 28 mai 2018 20:53
À : svpcom/wifibroadcast
Cc : baddemanax; Author
Objet : Re: [svpcom/wifibroadcast] RTL8723BE wifi card working (#14)


Have you use it as TX or RX?

**Comment by svpcom** [comment]
Yes, you could stream different steams parallel but total bandwidth should be less that max wifi bandwidth.
